Newer autoconf doesn't like us messing with LIBOBJS, and we're building
statically anyway, so just skip this step.
--- db-4.0.14/dist/configure.ac	2003-06-04 22:04:34.000000000 -0400
+++ db-4.0.14/dist/configure.ac	2003-06-04 22:04:29.000000000 -0400
@@ -451,8 +451,8 @@
 # Nasty hack.  AC_REPLACE_FUNCS added entries of the form xxx.o to the
 # LIBOBJS variable.  They have to be xxx.lo if we are building shared
 # libraries.  Use sed, configure already requires it.
-tmp="`echo \"$LIBOBJS\" | sed \"s/\.o/${o}/g\"`"
-LIBOBJS="$tmp"
+dnl tmp="`echo \"$LIBOBJS\" | sed \"s/\.o/${o}/g\"`"
+dnl LIBOBJS="$tmp"
 
 # Check for system functions we optionally use.
 AC_CHECK_FUNCS(_fstati64 getuid pstat_getdynamic sched_yield select strtoul)
